tests/keras_tests/feature_networks_tests/feature_networks/const_representation_test.py

```diff
@@ -81,6 +78,38 @@ def compare(self, quantized_model, float_model, input_x=None, quantization_info=
         self.unit_test.assertTrue(np.isclose(cs, 1), msg=f'fail cosine similarity check:{cs}')
 
 
+class ConstRepresentationMatMulTest(ConstRepresentationTest):
+    def __init__(self, unit_test, layer, const, is_list_input=False, input_reverse_order=False, use_kwrags=False,
```

Comment:
you don't need is_list_input & input_reverse_order
I think transpose_a is also irrelevant